DQMOffline/MuonDPG/src/CSCTnPEfficiencyTask.h

```diff
@@ -0,0 +1,67 @@
+#ifndef CSCTnPEfficiencyTask_H
+#define CSCTnPEfficiencyTask_H
+
+/*
+ * \file CSCTnPEfficiencyTask.h
+ *
+ * \author L. Lunerti - INFN Bologna
+ *
+*/
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
```

Comment:
Why is this deprecated header included here?

Reply:
Removing it (see [previous comment](https://github.com/cms-sw/cmssw/pull/36276/files/d5cceeb5969684a11b082de131ebabbe621d4210#r784757551))